Makefile

```
@@ -24,15 +24,15 @@ help-all: ## Display all help items, ie including plumbing targets
-	@OPA test policies
+	@OPA test policies data.json
 
 validate: ## Validate policy files
-	@opa check policies
+	@opa check policies data.json
 
 clean: # Cleanup build artifacts
 	@rm -f dist/*
 
 # Bundle the policies into a tarball for OCI registry
 build: clean ## Build the policy bundle
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)